### ISSUE TITLE
Support for multiple pod networks

### ISSUE THREAD

**Issue by Cerebus** [chaos-mesh/chaos-mesh#2247]
## Feature Request

Kubernetes can have multiple data planes connecting pods with multiple interfaces (multiple CNIs or instances of a CNI can run in parallel).   Each data plane connects to a pod as an additional interface (eth1 &etc).  It would be great to be able to apply network chaos on these networks.

**Comment by WangXiangUSTC** [comment]
So maybe we need to add a config for the network interface, does this meet your requirement？

**Comment by Cerebus** [comment]
Yes, as long as the experiment doesn't fail if the interface doesn't exist in a Pod.  The number of interfaces may vary across pods even in the same app.  

Configuring by qdisc by CIDR might be useful in addition (but not as a replacement).  Again, no failure if a selected Pod doesn't have an interface in that range.

**Comment by Cerebus** [comment]
Oh, and it should work w/ containerd.  Litmus experiments in this area rely on pumba, but pumba + containerd it giving me heartache.

**Comment by WangXiangUSTC** [comment]
Why pumba + containerd giving you heartache?

**Comment by YangKeao** [comment]
> Why pumba + containerd giving you heartache?

Because `Pumba` is chaos testing tool for Docker :smile_cat:

**Comment by YangKeao** [comment]
Are you using CNI-Genie to adapt multiple CNI plugin in the same kubernetes cluster?

**Comment by Cerebus** [comment]
I'm actually using [meshnet-cni](https://github.com/networkop/meshnet-cni) to run routed network emulations b/c it only emulates point to point links, which is 90% of what I need, but the Topology CRD is a pain to work with.  I started to look at multus.  I'll check out cni-genie.

I *think* multus will actually annotate Pods with all the attachment data.
